### PR TITLE
Update to 1.9.8

### PR DESCRIPTION
**Pull request by mpalmi**

### CODE REVIEW

File: 0.X/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM alpine:3.14
 
 # This is the release of Vault to pull in.
-ARG VAULT_VERSION=1.11.0
+ARG VAULT_VERSION=1.9.8
```
Comment (ccapurso) [inline 0.X/Dockerfile:4]: I have not actually reviewed a docker-vault PR before. This `VAULT_VERSION` change is for an older major version. This aligns with how we have updated this [in the past](https://github.com/hashicorp/docker-vault/commits/master/0.X/Dockerfile). Should I be approving these in a specific order?

Comment (mpalmi) [reply 0.X/Dockerfile:4]: Thanks for looking! The documented procedure is to merge and tag lowest to highest. I'll make sure to do that but approval doesn't necessarily have to happen in that order.

Comment (ccapurso) [reply 0.X/Dockerfile:4]: Great, thanks for the context!

**Review by ccapurso — APPROVED** [review]
Looks good

**Pull request merged**